common/src/main/java/com/wynntils/models/stats/StatListOrderer.java

```diff
@@ -18,104 +18,108 @@
 import java.util.Map;
 
 public final class StatListOrderer {
-    // Legacy order was defined by Athena as a fixed list. This was missing several stat types; I have
-    // tried filling them in into "logical" places
+    // Legacy order was defined by Athena as a fixed list.
+    // This was missing several stat types;
+    // I have tried filling them in into "logical" places.
+    // NOTE: This list was changed to use Artemis keys,
+    //       as a hotfix to support Item API V3.
+    //       This will be removed once the new encoding is implemented.
```

Comment:
I'm not sure we should remove it, even if it is not used for the chat. People might have been used to the way Wynntils Legacy ordered stats in the GUI, and want to keep it like that. Remember that this is the primary reason for the ordering of stats, the chat thing is mostly a side-effect.

Reply:
We can reconstruct that legacy ordering as actual stat type lists, so renaming stats won't break it.

Reply:
That sounds better.